plugins/postgresql/process-compose.yaml

```diff
@@ -0,0 +1,10 @@
+version: "0.5"
+
+processes:
+  postgresql:
+    command: "pg_ctl start -l $PGHOST/logfile -o \"-k $PGHOST\""
```

Comment:
You should be able to run postgres in foreground using `postgres` instead of `pg_ctl` I'm not entirely sure how the flags map.

Reply:
Tried this, but it has some weird issues when restarting (the process leaves behind a `postmaster.pid` file, which keeps postgres from restarting successfully.